src/legacy/core_plugins/data/public/search/aggs/agg_configs.ts

```diff
@@ -55,30 +50,56 @@ function parseParentAggs(dslLvlCursor: any, dsl: any) {
   }
 }
 
+export interface AggConfigsOptions {
+  schemas?: Schemas;
+  typesRegistry: AggTypesRegistryStart;
+}
+
+export type CreateAggConfigParams = Assign<AggConfigOptions, { type: string | IAggType }>;
```

Comment:
This type should be cleaned up in a follow-up PR so that the agg config params matches the returned interface from `AggConfig.toJson()`